src/Strategy/Node/YarnV1/YarnLock.hs

```diff
@@ -173,8 +175,8 @@ missingResolvedVersionErrorMsg key =
 pairToPackage :: YL.PackageKey -> YL.Package -> YarnV1Package
 pairToPackage key pkg = YarnV1Package (extractFullName key) (YL.version pkg)
 
```

Comment:
[suggest]: If we use a better extraction function like the suggested `extractPkgTriple`, then this can be removed:
```suggestion
```